virtcontainers/filesystem.go

```diff
@@ -438,6 +449,55 @@ func (fs *filesystem) sandboxURI(sandboxID string, resource sandboxResource) (st
 	return fs.resourceURI(true, sandboxID, "", resource)
 }
 
+func (fs *filesystem) sandboxesURI(resource sandboxResource) (string, string, error) {
+	var dirPath string
+
+	switch resource {
+	case configFileType:
+		dirPath = configStoragePath
+		break
+	case stateFileType, networkFileType, processFileType, lockFileType, mountsFileType, devicesFileType, hypervisorFileType, agentFileType, containersMapFileType:
```

Comment:
Are we going to handle all these kind of types at all?

Reply:
Nope, as I said to @jodh-intel, this was only a POC, I have to rework the code.

Reply:
done